### DISCUSSION TITLE
How can I transform Images from aces to bt.2020?

### DISCUSSION THREAD

**Discussion by frankC11** [OpenImageIO/oiio#3743] [Q&A]
Hey everyone, I'm new here and I don't know how to transform Images in another gamut. Is the command "colorconvert" the correct one? Can somebody help me please?

**Comment by lgritz** [comment]
The way we would usually do it is

    oiiotool in.exr -colorconvert aces bt2020 -o out.exr

Now, there are a number of assumptions:
* You have an OpenColorIO config with those names corresponding to the right color spaces.
* Your $OCIO env variable points to the config (or you also add the `--colorconfig /path/to/config` to the oiiotool command)
* The OpenImageIO package was build with OpenColorIO present, so that support was built in. (You can confirm that with `oiiotool --help` -- at the bottom there should be information about OCIO support and which config it found.)

**Reply by frankC11** [reply]
Thank you! I found in `oiiotool --help` only "linear bt.2020". Is there another gamma transformation? For example `gamma 2.2 bt.2020`

**Reply by lgritz** [reply]
The list of color spaces you see are no more and no less than whatever is in your OpenColorIO config.

**Reply by frankC11** [reply]
And what does the command `--ociodisplay`? Is it possible to combine this commands?

**Reply by lgritz** [reply]
Yes, you can chain together as many oiiotool commands as you want.

Docs on `--ociodisplay`:
https://openimageio.readthedocs.io/en/v2.4.7.1/oiiotool.html#cmdoption-ociodisplay

`oiiotool --help` will tell you all the displays and views that are available in your color configuration. (Or, for those readers building from the current master / in-progress 2.5, the command for this has changed to `oiiotool --colorconfiginfo`.)

**Reply by frankC11** [reply]
Thank you very much. What is the different between `--ociodisplay` and `--colorconvert` and do the known displays correspond to the ODTs?

**Reply by lgritz** [reply]
--colorconvert converts between two color spaces (given by name), and corresponds underneath to an OpenColorIO ColorConfig->getProcessor() call and applying it.

--ociodisplay corresponds underneath to an OpenColorIO DisplayViewTransform, and needs the names of the display, view, and optionally any looks to apply and the input color space.

I think if you need more detail about the differences, when these are appropriate, and exactly what they do in a color pipeline, it is probably better to ask on the OpenColorIO forums. They have a great Slack workspace with a "helpdesk" channel and are very responsive, with both the OCIO developers and the color scientists at several major studios quick to jump in and answer questions.
